backend/src/test/java/zipgo/auth/util/JwtProviderTest.java

```diff
@@ -0,0 +1,117 @@
+package zipgo.auth.util;
+
+import io.jsonwebtoken.Jwts;
+import org.junit.jupiter.api.*;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.test.context.SpringBootTest;
+import zipgo.auth.exception.AuthException;
+
+import java.util.Date;
+
+import static io.jsonwebtoken.SignatureAlgorithm.*;
+import static io.jsonwebtoken.security.Keys.*;
+import static java.nio.charset.StandardCharsets.*;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.junit.jupiter.api.Assertions.*;
+
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+@DisplayNameGeneration(DisplayNameGenerator.ReplaceUnderscores.class)
+@SuppressWarnings("NonAsciiCharacters")
+class JwtProviderTest {
+
+    @Autowired
+    private JwtProvider jwtProvider;
+
+    @Value("${jwt.secret-key}")
+    private String secretKey;
+
+    @Test
+    void 토큰을_생성한다() {
+        // given
+        String 페이로드 = String.valueOf(1L);
+
+        // expect
+        String 토큰 = jwtProvider.create(페이로드);
+
+        // then
+        assertThat(토큰).isNotNull();
+    }
+
+    @DisplayName("올바른 토큰 정보로 payload를 조회한다.")
+    @Test
+    void 올바른_토큰의_정보로_payload를_조회한다() {
+        // given
+        String 페이로드 = String.valueOf(1L);
+
+        // when
+        String 토큰 = jwtProvider.create(페이로드);
+
+        // then
+        assertThat(jwtProvider.getPayload(토큰)).isEqualTo(페이로드);
+    }
+
+    @Test
+    void 유효하지_않은_토큰의_형식으로_payload를_조회할_경우_예외가_발생한다() {
+        // expect
+        assertThatThrownBy(() -> jwtProvider.getPayload(null))
+                .isInstanceOf(AuthException.class);
+    }
+
+    @Test
+    void 만료된_토큰으로_payload를_조회할_경우_예외가_발생한다() {
+        // given
+        String 만료된_토큰 = Jwts.builder()
+                .signWith(hmacShaKeyFor(secretKey.getBytes(UTF_8)), HS256)
+                .setSubject(String.valueOf(1L))
+                .setExpiration(new Date((new Date()).getTime() - 1))
```

Comment:
```suggestion
        Date 과거_시간 = new Date((new Date()).getTime() - 1);
        String 만료된_토큰 = Jwts.builder()
                .signWith(hmacShaKeyFor(secretKey.getBytes(UTF_8)), HS256)
                .setSubject(String.valueOf(1L))
                .setExpiration(과거_시간)
```
이러면 가독성이 더 좋아질 것 같아요! 반영은 가비 맘~

Reply:
지금 다시 코드를 보니 잘 읽히지가 않네요~!

반영했어요~!